docs/tutorials/NMR/Examples_3_-Alanine_and_Silicates.md

```diff
@@ -0,0 +1,88 @@
+<h1>Examples 3 - Alanine and Silicates</h1>
+
+We now look at some more realistic examples. These are fairly large crystals - to get them
+to complete in a short time we will run them on a cluster.
+**Use the following number of
+cpus:**
+
+**analine - 4 cristoballite - 6 quartz - 13**
+
+(these numbers are chosen to give efficient k-point scaling)
+
+<p style="border-width:3px; border-style:solid;"><i>
+<b>Oxygen-17 NMR</b>
+<br>
+Oxygen is a component of many geological materials. Oxygen is
+also important element in organic and biological molecules since it is often intimately involved in hydrogen bonding. Solid State <sup><small>17</small></sup>O NMR should be a uniquely valuable probe as the chemical shift range of <sup><small>17</small></sup>O covers almost 1000 ppm in organic molecules. Furthermore <sup><small>17</small></sup>O has spin I = 5/2 and hence a net quadrupole moment. As a consequence of this the solid state NMR spectrum is strongly affected by the electric
+field gradient at the nucleus.
+
+Because the isotopic abundance of <sup><small>17</small></sup>O is very low (0.037%) and the NMR linewidths due to the electric field gradient relatively large, only limited Solid State NMR data is
+available. This is particularly true for organic materials. First principles calculations of ^17^O NMR parameters have played a vital role in assigning experimental spectra, and developing empirical rules between NMR  parameters and local atomic structure.</i>
+</p>
+
+# Alanine
+
+## Files
+
+* [alanine.cell](alanine/alanine.cell)
+* [alanine.param](alanine/alanine.param)
+* [alanine.pdb](alanine/alanine.pdb)
+
```

Comment:
Are we wanting to host these files locally on the docs?

Reply:
The files (apart from the param ones) are too large to be displayed like the previous files - not sure how else we could give them people, and I think this is how some other documentation did it

Reply:
That's fine, I just want to make sure. I can't resolve conversations on this repo, though.